clipping_manager/templates/clipping_manager/books.html

```diff
@@ -0,0 +1,75 @@
+{% extends "content.html" %}
+{% load i18n urlparams %}
+{% load staticfiles sekizai_tags %}
+
+{% block content %}
+    <div class="container mb-3 ">
+        <div class="row">
+            <div class="input-group col-10 pl-0">
+                <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-search" viewBox="0 0 16 16">
```

Comment:
Why `bi` and `bi-search`? What does the `bi` stand for in our context?

Reply:
These are probably abbreviations of "bootstrap icon". I don't think they are used anywhere, so I can safely remove them.